content/themes/semantic-ui/amazon/index.slim

```diff
@@ -9,8 +9,9 @@
       a.item data-tab="views" Views
       a.item data-tab="modules" Modules
 
-.ui.demo.container
+.ui.demo.container.basic.segment
```

Comment:
No, I can't definitely explained now. If not added, the right rail will go to the top of the page.